requirements.txt

```diff
@@ -2,7 +2,7 @@ h5py
 numpy>=1.16
 pandas
 nibabel
-pyyaml
+pyyaml>=5.3.1
```

Comment:
check the edu ch for this please.

Reply:
yes, I update this bcz the notebook has error